ras_party/controllers/party_controller.py

```diff
@@ -31,13 +31,17 @@ def parties_post(party_data, session):
         logger.info("Wrong sampleUnitType", type=party_data['sampleUnitType'])
         raise BadRequest(f'sampleUnitType must be of type {Business.UNIT_TYPE}')
 
-    business = query_business_by_ref(party_data['sampleUnitRef'], session)
+    sample_unit_ref = party_data['sampleUnitRef']
+    logger.info("about to create business", sample_unit_ref=sample_unit_ref)
+    business = query_business_by_ref(sample_unit_ref, session)
     if business:
         party_data['id'] = str(business.party_uuid)
         business.add_versioned_attributes(party_data)
+        logger.info("merging business", business=business)
```

Comment:
Isn't business for this line and the next just going to be the model?  I don't think the model has a sensible __repr__ won't it be something like <Business objerct at x0abc123>?